dsa-cp-1/Check-Anagrams/Explanation.md

```diff
@@ -0,0 +1,74 @@
+### Check whether two strings are anagram of each other 🙇
+
+▶️ Lets first understand what are anagrams. An anagram is a word or phrase formed by rearranging the letters in another word or phrase, such as spar, formed from rasp.
+
+▶️ In this problem we will check whether the strings which are given is anagram of each other or not.
+
+```
+1️⃣ Method
+
+▶️ First sort the given strings 
+▶️ Then compare the strings if they are same then return true or else fasle
+▶️ Time complexity : O(nlogn)
+```
+```c++
+
+bool checkAnagram(string s1, string s2)
+{
+    // If length of both strings is not same, then they
+    // cannot be anagram
+    if (s1.length() != s2.length())
+        return false;
+ 
+    // Sort both the strings
+    sort(s1.begin(), s1.end());
+    sort(s2.begin(), s2.end());
+ 
+    // Compare sorted strings
+    for (int i = 0; i < n1; i++)
+        if (s1[i] != s2[i])
+            return false;
+ 
+    return true;
+}
+```
+
+
+
+```
+2️⃣ Method
```

Comment:
Method 2 is incorrect.
Please validate with the below input.
ad and bc

Reply:
Thank you for pointing out the mistake I have updated the solution